src/core/evaluator.js

```diff
@@ -113,7 +113,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
     forceDataSchema: false,
     maxImageSize: -1,
     disableFontFace: false,
-    disableNativeImageDecoder: false,
+    nativeImageDecoderSupport: 'decode',
```

Comment:
Add `NativeImageDecoding` "enum" to src/shared/util.js and use NativeImageDecoding.DECODE (which will be equal to 'decode') here.